include/modmesh/SimpleArray.hpp

```diff
@@ -63,7 +63,9 @@ inline size_t buffer_offset(small_vector<size_t> const & stride, small_vector<si
     if (stride.size() != idx.size())
     {
         std::ostringstream ms;
+        // clang-format off
```

Comment:
There are some format that `clang-format` cannot handle well.  Use ``// clang-format off/on`` section to fence them.